FrontEnd/src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+import componentCSS from './Button.scss';
+
+class Button extends Component {
+
+
+    render() { 
+
+        console.log(this.props);
+
+    return ( <button onClick={() => this.props.onclick()} className={this.buttonClass()}>{this.props.Text}</button> );
+    };
+
+    buttonClass() {
```

Comment:
this is a personal thing, but I'd call this "getButtonClass" or "generateButtonClass" instead, so it's more obvious what it does